src/visualizers/VisualizerInterface.ts

```diff
@@ -60,4 +60,17 @@ export interface VisualizerInterface extends ParamableInterface {
      * @param element HTMLElement The DOM node the visualizer was inhabit()ing
      */
     depart(element: HTMLElement): void
+
+    /**
+     * Provides a way for visualizers to request a specific aspect ratio for
+     * its canvas. This aspect ratio is specified as a positive n > 0 where:
+     *  0 < n < 1:  The canvas is taller than it is wide
+     *  n = 1:      The canvas is a square
+     *  n > 1:      The canvas is wider than it is tall
+     * If the visualizer does not wish to request a specific aspect ratio and
+     * will instead work with whatever is given, this function may return
+     * `undefined` instead.
+     * @return the aspect ratio requested by the visualizer, or undefined if any
+     */
```

Comment:
This is a nice detailed comment. But given that there is already significant detail, please include the information that in Numberscope, as usual, aspect ratio is defined as width/height. Thanks.